scripts/flattener.py

```diff
@@ -1138,6 +1138,12 @@ def main(mfinal_id):
 				elif mxr['s3_uri'].endswith('h5ad'):
 					mxr_name = '{}.h5ad'.format(mxr_acc)
 				download_directory(mfinal_obj['spatial_s3_uri'], tmp_dir)
+				# If tissue_positions is present rename to tissue_positions_list and remove header
+				tiss_positions_check = os.path.exists(tmp_dir + '/spatial/tissue_positions.csv')
+				if tiss_positions_check == True:
```

Comment:
These 2 lines could be combined
`if os.path.exists(tmp_dir + '/spatial/tissue_positions.csv'):`

No strong feelings either way though